source/frontend/src/features/mapSideBar/acquisition/tabs/compensation/list/CompensationListView.tsx

```diff
@@ -52,13 +73,57 @@ export const CompensationListView: React.FunctionComponent<ICompensationListView
         }
       >
         <SectionField
-          label="Total payment amount for this file"
-          tooltip={`This is the total of all requisitions in the "Final" status.\nDraft entries are not included here.`}
-          labelWidth="9"
+          label={
+            <>
+              <FaHandHoldingUsd size={24} className="mr-4" />
+              Total allowable compensation
+            </>
+          }
+          tooltip={`This is the maximum allowable for this file. Edit to set or change this value.`}
```

Comment:
nit: please use straight strings here (a bit more readable) - ie tooltip="This is the maximum..."